commands/sandbox_util.go

```diff
@@ -0,0 +1,183 @@
+/*
+Copyright 2018 The Doctl Authors All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package commands
+
+import (
+	"crypto/sha1"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/digitalocean/doctl/do"
+)
+
+const (
+	// Minimum required version of the sandbox plugin code.  The first part is
+	// the version of the incorporated Nimbella CLI and the second part is the
+	// version of the bridge code in the sandbox plugin repository.
+	minSandboxVersion = "4.1.0-1.3.0"
+
+	// The version of nodejs to download alongsize the plugin download.
+	nodeVersion = "v16.13.0"
```

Comment:
Doesn't look like the `nodeVersion` const from the commands package is being used.

```
$ grep -r nodeVersion .
./commands/sandbox_util.go:     nodeVersion = "v16.13.0"
./do/sandbox.go:        nodeVersion = "v16.13.0"
./do/sandbox.go:                nodeDir = fmt.Sprintf("node-%s-%s-%s", nodeVersion, goos, arch)
./do/sandbox.go:                nodeURL = fmt.Sprintf("https://nodejs.org/dist/%s/%s.tar.gz", nodeVersion, nodeDir)
./do/sandbox.go:                        nodeURL = fmt.Sprintf("https://nodejs.org/dist/%s/%s.zip", nodeVersion, nodeDir)
./do/sandbox.go:                return installed == nodeVersion
```

Reply:
Right.  Fixed.